### PR TITLE
Integrate Sprint 5 of Team 1

### PR DESCRIPTION
**Pull request by Erdnaf**
This is the pull request for the work of Team `#1` in Sprint `#5`.

### CODE REVIEW

File: privacy_evaluator/attacks/membership_inference/on_point_basis.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        x_train: np.ndarray,
+        y_train: np.ndarray,
+        x_test: np.ndarray,
+        y_test: np.ndarray,
+    ):
+        """Initializes a MembershipInferenceOnPointBasis class.
+
+        :param target_model: Target model to be attacked.
+        :param x_train: Data which was used to train the target model.
+        :param y_train: True, one-hot encoded labels for `x_train`.
```
Comment (fraboeni) [inline privacy_evaluator/attacks/membership_inference/on_point_basis.py:22]: What do you mean by true here? Isn't it a vector with the labels only and not a Boolean value?

File: privacy_evaluator/attacks/membership_inference/on_point_basis.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        :param x_test: Data that was not used to train the target model.
+        :param y_test: True, one-hot encoded labels for `x_test`.
```
Comment (fraboeni) [inline privacy_evaluator/attacks/membership_inference/on_point_basis.py:24]: Same here.

**Review by fraboeni — CHANGES_REQUESTED** [review]

**Comment by fraboeni** [comment]
Apart from the small question to the documentation in the one function everything looks good to me. Thanks for your efforts!

**Comment by Friedrich-Mueller** [comment]
I quickly removed these 'True's from 5 files total within the membership_inference folder. If I'm not mistaken, these were the only locations where they occurred.

I think we can merge now?

**Pull request merged**